### PR TITLE
fastfec 0.1.6

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/fastfec.rb [hunk 1]
```diff
@@ -21,10 +21,7 @@ class Fastfec < Formula
 
   def install
     ENV["ZIG_SYSTEM_LINKER_HACK"] = "1"
-    args = [
-      # Use brew's pcre
-      "-Dvendored-pcre=false",
-    ]
```
Comment (SMillerDev) [inline Formula/fastfec.rb:27]: Does that mean it vendors pcre now?

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/fastfec.rb (hunk 1 above)
Comment (alebcay) [reply Formula/fastfec.rb:27]: Based on https://github.com/washingtonpost/FastFEC/pull/35, sounds like they vendor PCRE sources now and don't include the option out of the box to link to a system installation of PCRE.

**Comment by chenrui333** [comment]
Gonna reopen after the upstream issues get resolved.

**Comment by chenrui333** [comment]
cc @freedmand, can you help with the reported issues? Thanks!

File: Formula/fastfec.rb [hunk 2]
```diff
@@ -21,10 +21,7 @@ class Fastfec < Formula
```
Comment (freedmand) [inline Formula/fastfec.rb:21]: We can remove `uses_from_macos "curl"` as it's no longer a dependency

File: Formula/fastfec.rb [hunk 3]
```diff
@@ -35,7 +32,7 @@ def install
   end
 
   test do
-    system "#{bin}/fastfec", "--no-stdin", "13425"
+    system("curl https://docquery.fec.gov/dcdev/posted/13425.fec | #{bin}/fastfec 13425")
```
Comment (freedmand) [inline Formula/fastfec.rb:35]: If `curl` is no longer a dependency, will these tests still work? Is there a way to specify curl as a test-only dep?

File: Formula/fastfec.rb (hunk 1 above)
Comment (freedmand) [reply Formula/fastfec.rb:27]: Per https://github.com/washingtonpost/FastFEC/pull/46 we will be able to keep this flag value

**Review by freedmand — COMMENTED** [review]
Some light questions on Curl. I also PRd a fix to the vendored pcre problem: https://github.com/washingtonpost/FastFEC/pull/46

**Comment by freedmand** [comment]
cc @chenrui333, thank you so much for opening this up. I PRd a fix to the vendored issue https://github.com/washingtonpost/FastFEC/pull/46

**Comment by freedmand** [comment]
I will say, I am worried per https://github.com/washingtonpost/FastFEC/issues/16, whether Zig will prevent the vendored pcre from actually being used

**Pull request closed without merging**